utils/js/getJson.js

```diff
@@ -0,0 +1,10 @@
+export async function fetchData(url) {
+  try {
+    const response = await fetch(url);
+    const data = await response.json();
+    return data;
+  } catch (err) {
+    console.log("Error : ", err);
```

Comment:
에러를 콘솔로 찍는 경우에 error로도 표현할 수 있어요
```suggestion
    console.error("Error : ", err);
```

Reply:
아 그렇군요 ,,!! 에러를 콘솔로 찍어볼 땐 좀 더 직관적으로 알아볼 수 있도록 console.error을 사용해봐야겠어요 !!